### PR TITLE
update to OPLS, RMSD, examples, and make everything python3 compatible

### PR DESCRIPTION
**Pull request by bannanc**
I've made a few minor changes to smi2sdf, genMOL2, and oe_min_openMM to make them python3 compatible. I also add one more error check to oe_min_openMM. None of these changes should affect behavior.

### CODE REVIEW

**Comment by bannanc** [comment]
@vtlim This includes a bunch of updates to OPLS.py and RMSD.py

Also all of the scripts are python3 compatible and the examples directory has an example run with all of the scripts. 

Note - I updated RMSD to allow for multiple forcefields as the --ref input, but I imagine with the larger datasets we will still want to use 1 and make separate output files since otherwise there will be way too much data to deal with.

**Comment by bannanc** [comment]
Oh, its actually ready for review now

**Comment by bannanc** [comment]
@nthi0625 
You should take a look at this, I updated the opls and rmsd scripts you had in your PR so I could use them on a few weird molecules

File: OPLS.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 #!/usr/bin/env python
 
 ### Author:
-# Daisy Y. Kyu
 # Nam Thi
-# Victoria Lim (limvt@uci.edu)
 # Caitlin C. Bannan (bannanc@uci.edu)
+# Daisy Y. Kyu
```
Comment (vtlim) [inline OPLS.py]: I don't think Daisy worked on this script?

File: OPLS.py [hunk 2]
```diff
@@ -26,58 +26,136 @@
...
             type = "string",
-            dest = 'opttype')
+            dest = 'dir3')
+
     (opt, args) = parser.parse_args()
-    OPLSMin(opt.ifile,opt.opttype)
+    # Check input directory
+    if opt.idir is None:
+        parser.print_help()
+        parser.error("ERROR: you must provide an input file")
```
Comment (vtlim) [inline OPLS.py]: this should say input directory

File: RMSD.py [hunk 3]
```diff
@@ -7,7 +7,7 @@
 # Caitlin C. Bannan (bannanc@uci.edu)
```
Comment (vtlim) [inline RMSD.py]: I'm not sure that Daisy worked on this script either? We could double check.

**Comment by vtlim** [comment]
@bannanc Everything looks great to me. Unless you want to edit anything else, I'll fix some typos / grammar stuff, then can merge changes.

**Comment by vtlim** [comment]
@bannanc, these scripts now supersede #43, correct? If so, I'll close without merging that pull request.

**Comment by bannanc** [comment]
Oh, yeah I didn't see that before, but yes this should supersede #43. It should also supersede #37 if you're happy with my code changes

File: OPLS.py (hunk 1 above)
Comment (bannanc) [inline OPLS.py]: I didn't actually add her, she was already on the list, I just changed the order of authors.

File: RMSD.py (hunk 3 above)
Comment (bannanc) [inline RMSD.py]: She was already listed, I assumed Nam must have worked with her or something

**Comment by bannanc** [comment]
@vtlim I fixed the things you pointed out in your comments. And add the things we talked about in our meeting today. 

However, when I tried to put `automorph=True` or `heavyOnly=True` into the `OERMSD(...)` I got a warning like 
````
TypeError: OERMSD() got an unexpected keyword argument 'overlay'
```
So I put a comment above the call instead saying we set all three of those things to True, if it is an optional variable I don't understand why you can't state which one you're setting. It might have something to do with openeye being coded in C++ then having a python wrapper.

**Pull request merged**